cfgen/cfgen

```diff
@@ -892,17 +907,17 @@ class LibfabricEndpoint(Endpoint):
     ep_map: Dict[Tuple[str, int], int] = {}
 
     def __init__(self, NetFamily: str, proto: NetProtocol, ipaddr: str,
-                 portalgroup: LibfabPortalGroup):
+                 portalgroup: int):
```

Comment:
If we define classes as above, we can then use the class type here and this will change as,

def __init__(self, NetFamily: str, proto: NetProtocol, ipaddr: str, portalgroup: LibfabricPortalGroup, proct: str)